packages/dart/noports_core/lib/src/srv/srv_impl.dart

```diff
@@ -630,69 +637,111 @@ class SrvImplDart implements Srv<SocketConnector> {
     return socketConnector;
   }
 
-  Future _handleMultiConnectRequest(
+  Future<void> _handleMultiConnectRequest(
+    SocketConnector sc,
+    InternetAddress relayAddress,
+    DataTransformer? encrypter,
+    DataTransformer? decrypter,
+  ) async {
+    logger.info('_runDaemonSideMulti'
+        ' Control socket received connect request - '
+        ' creating new socketToSocket connection');
+
+    InternetAddress localAddress = await resolveRequestedLocalHost();
+
+    // First, connect to the relay.
+    // If it fails, return, as we can't do anything more
```

Comment:
Note that once we have connected and authenticated to the relay, the relay will join it to the client-side.